nautobot/core/views/generic.py

```diff
@@ -260,8 +262,10 @@ def get(self, request):
                     request.GET, self.filterset
                 )
                 dynamic_filter_form = DynamicFilterFormSet(filterset_class=self.filterset, data=factory_formset_params)
+                filter_form = self.filterset_form(filter_params, label_suffix="")
```

Comment:
Thinking towards future removal of filterset_forms, can we make this so that it handles things gracefully (i.e. renders without the tab at all) if `self.filterset_form` is None?